src/PrestaShopBundle/Resources/views/Admin/Module/tab-modules-list.html.twig

```diff
@@ -70,3 +70,23 @@
     <a href="https://addons.prestashop.com/?utm_source=back-office&amp;utm_medium=dispatch&amp;utm_campaign=back-office-{{ app.request.locale }}&amp;utm_content=download{% if adminListFromSource is defined %}&amp;utm_term={{ adminListFromSource }}{% endif %}" onclick="return !window.open(this.href);">{{ 'More modules on addons.prestashop.com'|trans({}) }}</a>
```

Comment:
No domains assigned for the three strings: Not Installed, Installed, More modules on addons.prestashop.com.